docs/src/gallery/guides.md

```diff
@@ -0,0 +1,101 @@
+# Guides
+
+## Guide.annotation
```

Comment:
These should be ideally be links, e.g. `[Guide.annotation](@ref)`

Reply:
i've added links from the gallery to the library in the most recent commit.  thanks for the feedback!